public/app/plugins/panel/singlestat/module.ts

```diff
@@ -300,7 +303,7 @@ class SingleStatCtrl extends MetricsPanelCtrl {
     this.setValueMapping(data);
   }
 
-  setValueMapping(data) {
+  setValueMapping(data: { value: any; valueFormatted: any; valueRounded: any }) {
```

Comment:
This type annotation is repeated. Can you extract that to a separate interface (and keep the definition in this file?). Note that it's better not to name it `ValueMapping` as we already have an interface like that used for value mapping editor  (packages/grafana-ui/src/types/panel.ts)

Reply:
Should I go for the same types on the interface as here, or is `any` sufficient?

https://github.com/grafana/grafana/pull/17616/files/2a869af20da45073555e3773e4c6f5c0bbd9a457#diff-e593190b57ddfe3f482e43e7b8248993R184

Reply:
I would opt for specific type, especially that you have created it's basic shape already :)